### DISCUSSION TITLE
Welcome to seckillcloud Discussions!

### DISCUSSION THREAD

**Discussion by weiraneve** [weiraneve/seckillcloud#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mengweiyu** [comment]
Can u add Nginx and anything cool?

**Reply by weiraneve** [reply]
This is a well idea and I will achieve it.
